contrib/toxic/patches/makefile.patch

```diff
@@ -0,0 +1,39 @@
+--- a/Makefile
++++ b/Makefile
+@@ -5,11 +5,11 @@
+ 
+ LIBS = toxcore ncursesw libconfig libcurl
+ 
+-CFLAGS ?= -std=c11 -pthread -Wall -Wpedantic -Wunused -fstack-protector-all -Wvla -Wno-missing-braces -Wmissing-prototypes -Wcast-align -Wcast-qual -Wmissing-declarations -Wshadow -Wunused-macros -Wformat-nonliteral -Wno-unknown-pragmas
++CFLAGS = -std=gnu11 -pthread -Wall -Wpedantic -Wunused -fstack-protector-all -Wvla -Wno-missing-braces -Wmissing-prototypes -Wcast-align -Wcast-qual -Wmissing-declarations -Wshadow -Wunused-macros -Wformat-nonliteral -Wno-unknown-pragmas
+ CFLAGS += '-DTOXICVER="$(VERSION)"' -DHAVE_WIDECHAR -D_XOPEN_SOURCE_EXTENDED -D_FILE_OFFSET_BITS=64
+ CFLAGS += '-DPACKAGE_DATADIR="$(abspath $(DATADIR))"'
+ CFLAGS += ${USER_CFLAGS}
+-LDFLAGS ?=
++LDFLAGS =
```

Comment:
this doesn't do anything whatsoever as it's already correct

Reply:
My bad I had issue before adding `make_use_env` and so I tried the alpine's patches, I forgot to remove it